### PR TITLE
[DDW-821] Update packages with vulnerabilities

### PR DESCRIPTION
**Pull request by przemyslaw-wlodek**
This PR aims to get rid of some codebase vulnerabilities by bumping up the version of dependencies.

## Todos

- N/A

## Screenshots

- N/A

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1638436752339100)
-  [Testing report ](https://docs.google.com/spreadsheets/d/16gxSHXpTW5dkEC7BEXdaxcdVQwsTkwYtcOhSiAn5UVg/edit#gid=0)

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes
- [ ] PR has default-sized Daedalus window screenshots or animated GIFs of important UI changes:
  - [ ] In English
  - [ ] In Japanese
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: package.json
Comment (danielmain) [inline package.json:113]: in confluence we have here 2.3.5 which one is correct?

**Review by danielmain — CHANGES_REQUESTED** [review]
Great job! just one package has the wrong version. check the comments pls.

File: package.json
Comment (przemyslaw-wlodek) [reply package.json:113]: Unfortunately `2.3.5` does not work. Let me update the confluence document. Thank you!

**Review by mchappell — APPROVED** [review]
👍

**Review by dmitrii-gaico — APPROVED** [review]
The app looks stable and no obvious issues detected. However let's do not merge this yet until we have more reviews from QA side.
cc @input-output-hk/daedalus-qa @przemyslaw-wlodek

**Review by danielmain — APPROVED** [review]
Looks great @przemyslaw-wlodek.

**Pull request merged**
